pandas/tests/indexes/common.py

```diff
@@ -332,7 +332,7 @@ def test_memory_usage(self, indices):
         # RangeIndex, IntervalIndex
         # don't have engines
         if not isinstance(indices, (RangeIndex, IntervalIndex)):
-            assert result2 > result
```

Comment:
is it just PeriodIndex for which this inequality is no longer strict?

Reply:
@jbrockmendel 
Yes. for PeriodIndex